src/applications/gi/components/profile/CalculatorForm.jsx

```diff
@@ -45,6 +45,25 @@ class CalculatorForm extends React.Component {
     return extensions;
   }
 
+  createExtensionOption = extension => {
+    const extensionOption = {
+      value: extension.zip,
+      label: '',
+    };
+
+    extensionOption.label = extension.institution;
```

Comment:
Since `extensionOption.label` is going to start as this value anyway, we could initialize above: 

```
label: extension.instituion,
```

Reply:
Changes are added. Thanks for the feedback!